l5kit/l5kit/evaluation/extract_ground_truth.py

```diff
@@ -21,25 +19,22 @@ def export_zarr_to_ground_truth_csv(
     Arguments:
         zarr_dataset (np.ndarray): The open zarr dataset.
         csv_file_path (str): File path to write a CSV to.
-        history_num_frames (int): Amount of history frames to draw into the rasters.
-        future_num_frames (int): Amount of history frames to draw into the rasters.
+        future_num_frames (int): Amount of future displacements we want.
         filter_agents_threshold (float): Value between 0 and 1 to use as cutoff value for agent filtering
-        pixel_size (np.ndarray) -- Size of one pixel in the real world
         history_step_size (int): Steps to take between frames, can be used to subsample history frames.
         future_step_size (int): Steps to take between targets into the future.
-        based on their probability of being a relevant agent.
     """
 
     assert future_step_size == history_step_size == 1, "still not handled in select_agents"
     cfg = {
         "raster_params": {
-            "pixel_size": pixel_size,
+            "pixel_size": np.asarray((0.25, 0.25)),
             "raster_size": (100, 100),
             "filter_agents_threshold": filter_agents_threshold,
             "ego_center": np.asarray((0.5, 0.5)),
         },
         "model_params": {
-            "history_num_frames": history_num_frames,
+            "history_num_frames": 0,
```

Comment:
Do we actually need to create a cfg with some unused parameters or could we just pass them to the constructur as is?

Reply:
trouble here is that the partial inside read the configuration, so we need to pass a kind-of valid config..I agree it sucks though xD